.template/addons/nginx/config/environments/production.rb

```diff
@@ -0,0 +1,11 @@
+gsub_file(
+  'config/environments/production.rb',
+  "config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?",
+  'config.public_file_server.enabled = false',
+)
+
+gsub_file(
+  'config/environments/production.rb',
+  "config.public_file_server.enabled = true",
```

Comment:
Please update the quote to a single quote to make it consistent with the others.

Reply:
Fixed in https://github.com/nimblehq/rails-templates/pull/232/commits/40fe6a21861ff609f5f99ade75cd0958daaacc46 @byhbt